src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -88,12 +86,17 @@ private int Run(string[] args)
                     }
                     else if (ArgumentMatch(arg, "assembly") || ShortNameArgumentMatch(arg, "a"))
                     {
-                        if (assembly != null)
+                        if (assembly != null || ++i >= args.Length)
```

Comment:
If I have `sgen foo.dll -a bar.dll -quiet`, do we want to report 1 error for invalid "-a" argument or report 2 errors with the second error for invalid argument "bar.dll"?

Reply:
@zhenlan It makes more sense to me to report two errors. (It is reporting two errors right now, the second error will be added from line 121)

Reply:
Will it be confusing if an error is saying bar.dll is an invalid argument while it follows exactly as the usage formatting?

Reply:
We only support one value for assembly parameter e.g. sgen foo.dll. If to generate two , you need call sgen twice.

Reply:
I am just giving an example of error case.